src/test/java/TestsWithRestAssured/BoardForTests.java

```diff
@@ -0,0 +1,35 @@
+package TestsWithRestAssured;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BoardForTests {
+
+    private String name;
+    private String desc;
+    private boolean closed;
+
+    public String getName() {
```

Comment:
You can use Lombok library to generate getters and setters with just one annotation. Or - starting from Java14 - You can turn this class into a record